.cmake/project-config.cmake.in

```diff
@@ -1,6 +1,9 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
-find_dependency(spdlog @SPDLOG_MIN_VERSION@ REQUIRED COMPONENTS spdlog)
+#
+# TODO: find_dependency(spdlog @SPDLOG_MIN_VERSION@ REQUIRED COMPONENTS spdlog)
```

Comment:
this should be addressed!

see https://discourse.cmake.org/t/how-to-install-component-libraries-like-boost/2854/2

Reply:
done